enzyme/test/ActivityAnalysis/integration.ll

```diff
@@ -0,0 +1,93 @@
+; RUN: if [ %llvmver -gt 15 ]; then %opt < %s %newLoadEnzyme -passes="print-activity-analysis" -activity-analysis-func=f.preprocess-S | FileCheck %s; fi
```

Comment:
change this to OPnewLoadEnzyme instead of newLoadEnzyme so you have opaque pointer support?

Reply:
I think the printer is not compatible with opaque pointers before llvm 17:

```
#if LLVM_VERSION_MAJOR >= 17
#else
      auto et = a.getType()->getPointerElementType();
      if (et->isFPOrFPVectorTy()) {
        dt = TypeTree(ConcreteType(et->getScalarType())).Only(-1, nullptr);
      } else if (et->isPointerTy()) {
        dt = TypeTree(ConcreteType(BaseType::Pointer)).Only(-1, nullptr);
      }
```

Reply:
I have tried to add some logic to avoid that if there is an opaque pointer type